dist/frappe-charts.esm.js

```diff
@@ -130,9 +122,6 @@ function getExtraWidth(m) {
 const INIT_CHART_UPDATE_TIMEOUT = 700;
 const CHART_POST_ANIMATE_TIMEOUT = 400;
 
-const DEFAULT_AXIS_CHART_TYPE = 'line';
-const AXIS_DATASET_CHART_TYPES = ['line', 'bar'];
-
```

Comment:
This may be problematic. I searched the usage for these 2 constants:
https://github.com/frappe/charts/blob/master/src/js/utils/axis-chart-utils.js#L41

I don;t really understand what that line does. Seems to be a bug there, should be `=` not `===`? @pratu16x7